src/app/components/Byline/index.test.tsx

```diff
@@ -110,22 +110,6 @@ describe('Byline', () => {
     expect(timestamp).toBeInTheDocument();
   });
 
-  it('should correctly render an extra listitem for Timestamp', () => {
```

Comment:
Has this test been moved elsewhere?

Reply:
no, this was mistakenly removed! my bad, i must have done this when it was failing as its now no longer a list item, although while thinking about a11y stuff this seems incorrect - will rethink the html semantics .